frontend/src/pages/sign-up.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+class SignUpPage extends React.Component {
```

Comment:
rather than extending `React.Component` we use the functional component convention (at least in the time I've been here). So this component could be rewritten as 

```
const SignUpPage = () => {
  return (
    <div>
      <h1>Sign Up Page</h1>
    </div>
  );
};

export default SignUpPage;
```

Reply:
same with all the other components

Reply:
ok.. guess that goes for all of them